src/pages/kb/data-sources/bigquery-setup.md

```diff
@@ -8,15 +8,15 @@ toc: true
 
 ### Data Source Setup Screen
 
-To set up the BigQuery data source only the **Project ID** and **JSON Key File** are required by Redash. Occasionally, BigQuery will require that you include a **Processing Location** as well. You can get a key file when you create a new service account with Google (details below). 
+On the BigQuery Data Source setup screen, the **Project ID** and **JSON Key File** are always required.  You can get a key file when you create a new service account with Google (details below). 
 
 ![](/assets/images/docs/gitbook/bigquery_mandatories.PNG)
 
 + If your database schema exceeds \~2500 tokens, untick the **Load Schema** box to stop Redash from loading it to the Query Editor screen. Many browsers will slow down or crash if the schema is too big.
 
-+ Since BigQuery 2.0, you can use either BigQuery SQL syntax (now called Legacy SQL) or [Standard SQL Syntax](https://cloud.google.com/bigquery/docs/reference/standard-sql/migrating-from-legacy-sql). Redash supports both, but Standard SQL is the default.  This preference applies at the Data Source-level by toggling the **Use Standard SQL** box. Your selection here is passed to BigQuery along with your query text. If some of your queries use Legacy SQL and others use Standard SQL, you can create two data sources.
++ Since BigQuery 2.0, BigQuery supports SQL syntax (now called Legacy SQL) or [Standard SQL Syntax](https://cloud.google.com/bigquery/docs/reference/standard-sql/migrating-from-legacy-sql). Redash supports both, but Standard SQL is the default.  This preference applies at the Data Source-level by toggling the **Use Standard SQL** box. Your selection here is passed to BigQuery along with your query text. If some of your queries use Legacy SQL and others use Standard SQL, you can create two data sources.
```

Comment:
```suggestion
+ Since BigQuery 2.0, BigQuery supports its Legacy SQL syntax or [Standard SQL Syntax](https://cloud.google.com/bigquery/docs/reference/standard-sql/migrating-from-legacy-sql). Redash supports both, but Standard SQL is the default.  This preference applies at the Data Source-level by toggling the **Use Standard SQL** box. Your selection here is passed to BigQuery along with your query text. If some of your queries use Legacy SQL and others use Standard SQL, you can create two data sources.
```